### PR TITLE
Added basic class structure 

### PR DESCRIPTION
**Pull request by KshitijAggarwal**
Added Observation and telescope classes and their tests.

### CODE REVIEW

File: arts/telescope.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+
+class Telescope:
+    """
+    Class with Telescope properties.
+
+    """
+
+    def __init__(self):
```
Comment (devanshkv) [inline arts/telescope.py:10]: Maybe add the name, location, etc to the constructor arguments?

File: arts/observation.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+from arts.telescope import Telescope
+
+
+class Observation(Telescope):
```
Comment (devanshkv) [inline arts/observation.py:4]: Maybe the Observation should take an instance or list of instances of the Telescope class?

This way, we can run a single observation for multiple telescopes?

**Review by devanshkv — CHANGES_REQUESTED** [review]

File: arts/telescope.py (hunk 1 above)
Comment (KshitijAggarwal) [reply arts/telescope.py:10]: Good idea. Will do.

Comment (KshitijAggarwal) [reply arts/telescope.py]: Done.

**Review by devanshkv — APPROVED** [review]

**Pull request merged**